src/java/org/apache/cassandra/index/sai/IndexContext.java

```diff
@@ -406,7 +406,7 @@ public String logMessage(String message)
      * @return the indexes that are built on the given SSTables on the left and corrupted indexes'
      * corresponding contexts on the right
      */
-    public Pair<Collection<SSTableIndex>, Collection<SSTableContext>> getBuiltIndexes(Collection<SSTableContext> sstableContexts, boolean validate)
+    public Pair<Collection<SSTableIndex>, Collection<SSTableContext>> getBuiltIndexes(Collection<SSTableContext> sstableContexts, IndexValidation validate)
```

Comment:
Nit: I'd call the parameter `validation`, instead of the current boolean-style `validate`

Reply:
fixed